internal/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bazel/typescript",
+  "description": "Custom TypeScript compiler for hosting on Bazel",
+  "homepage": "https://github.com/bazelbuild/rules_typescript",
+  "license": "Apache-2.0",
+  "version": "0.0.0-PLACEHOLDER",
+  "keywords": [
+      "typescript",
+      "bazel"
+  ],
+  "bin": {
```

Comment:
maybe there's not much point advertising this as a bin
if you run it outside bazel it fails because of worker something
if you run it inside bazel you don't use this nodejs_binary

Reply:
agreed

Reply:
done